packages/serialize/src/index.ts

```diff
@@ -203,30 +283,39 @@ function handleInterpolation(
         )
         if (matched.length) {
           console.error(
-            '`keyframes` output got interpolated into plain string, please wrap it with `css`.\n\n' +
-              'Instead of doing this:\n\n' +
-              [...matched, `\`${replaced}\``].join('\n') +
-              '\n\nYou should wrap it with `css` like this:\n\n' +
-              `css\`${replaced}\``
+            `\`keyframes\` output got interpolated into plain string, please wrap it with \`css\`.
+
+Instead of doing this:
+
+${[...matched, `\`${replaced}\``].join('\n')}
+
+You should wrap it with \`css\` like this:
+
+css\`${replaced}\``
```

Comment:
I rewrote this string interpolation as semantic coloring broke on VSCode due to the complex ' and ` usage
![Screen Shot 2021-11-08 at 17 10 03](https://user-images.githubusercontent.com/17657014/140937927-1ab0bf83-385d-4e17-b50c-6f8368fd44cd.png)